### DISCUSSION TITLE
Checking dependency resolution for another Maven example project

### DISCUSSION THREAD

**Discussion by Markus-Elfring** [remkop/picocli#1776] [Q&A]
I added [the mentioned dependency settings](https://github.com/remkop/picocli/blob/dbf907e1b42ddd36711f6624fc822e7adca75eaa/README.md#maven "Dependency information") to the file “pom.xml” which should be managed by software components of a known development environment.

Eclipse IDE for Java Developers
Version: 2022-06 (4.24.0)
Build id: 20220609-1112

I assume that files which are required for the desired CLI tool should automatically be retrieved from a corresponding Maven repository.
Unfortunately, I stumbled on the error message “The import picocli cannot be resolved” during my collaboration check for the involved development tools.

I would appreciate advices accordingly.

**Comment by remkop** [comment]
Sorry, I have no idea where the problem could be.
Maybe start from a working project, make a copy and modify that?
Also, this seems more like related to the Eclipse IDE than to picocli.
Maybe the Eclipse IDE's user manual has useful tips?

**Reply by Markus-Elfring** [reply]
>Also, this seems more like related to the Eclipse IDE than to picocli.

:thought_balloon: If you would like to reuse a known dependency specification also for this software, it seems that the circumstances become more interesting under which the desired build configuration does not work as expected.
I found a few experience (or bug) reports which indicate opportunities for further development considerations according to other version combinations of involved components for this issue. :thinking:

**Reply by remkop** [reply]
Okay, best of luck! 🤞

**Reply by Markus-Elfring** [reply]
:thought_balloon: Would you like to add any ideas or views to [another related clarification attempt](https://github.com/apache/netbeans/issues/4563 "Message “package … does not exist” despite of a corresponding Java source file")?
